templates/taxi/car_form.html

```diff
@@ -9,4 +9,4 @@ <h1>{{ object|yesno:"Update,Create" }} car</h1>
 
     <input type="submit" value="Submit" class="btn btn-primary">
```

Comment:
According to the checklist, you should add a 'Cancel' button that would lead to the previous page the user was on. You could add it next to the 'Submit' button.